snuba/datasets/events.py

```diff
@@ -200,6 +213,23 @@ def get_promoted_context_columns(self):
     def get_required_columns(self):
         return self.__required_columns
 
+    def column_expr(self, column_name, body):
+        if column_name in TIME_GROUP_COLUMNS:
```

Comment:
Since you are now only processing TIME_GROUP_COLUMNS in the events dataset, I would advise to move everything here:
- time_expr method
- the setting (TIME_GROUP_COLUMNS). Which at this point can be hardcoded into the events dataset. Should this become useful for other datasets in the future it will be easy to move, but as long as time groups are not processed outside events, it is better to keep them coupled with the events dataset itself.

Or, as an alternative, I am ok with making TIME_GROUP_COLUMNS generic, but then this line should go back to util.py

Reply:
I moved this back into util.py - I'm not sure if this will be in every dataset in future but it seems somewhat generically useful for now